tests/dragonfly/replication_test.py

```diff
@@ -1971,23 +1971,27 @@ async def test_replicaof_reject_on_load(df_factory, df_seeder_factory):
     df_factory.start_all([master, replica])
 
     c_replica = replica.client()
-    await c_replica.execute_command(f"DEBUG POPULATE 8000000")
+    await c_replica.execute_command(f"DEBUG POPULATE 4000000")
```

Comment:
The only reason we use large amount of keys here is to generate lots of data which will be later take time to load.
If you will use 
seeder = StaticSeeder(
        key_target=800,
        data_size=100000,
        collection_size=10000,
        types=["SET"])
The test will be extremely fast and this will be same data size.
2 reasons for this:
1. when populating we yeild every 32 keys added I think
2. we add multiple elements in one command when using collections while for string we each time add one key

Reply:
in that case the variation of "debug populate ... type SET ... " could also work.

Reply:
i just confirmed `debug populate 800 key 1000 RAND type set elements 50000` creates 800 sets with 50000 elements of size 1000 each

Reply:
StaticSeeder uses debug poulate its similar to your fix, anyway the change is good